theme/icons/file-audio.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This image is smaller than the others, not sure why or how. Look at them blown up in the fyne_demo Graphics tab